404.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
```

Comment:
Should we have a meta-redirect here as well (perhaps with a 1..2 seconds timeout)?;

```html
<meta http-equiv="refresh" content="1; url=https://docs.docker.com" />
```

Reply:
I put it for 1 second as you suggested; should be enough time for JS to kick in if enabled.